tools/src/main/scala/scala/scalanative/build/Build.scala

```diff
@@ -69,6 +69,6 @@ object Build {
       LLVM.compile(config, generated)
     }
 
-    LLVM.link(config, linked, objectFiles, unpackedLibs, outpath)
+    LLVM.link(config, linked, objectFiles, outpath)
```

Comment:
`unpackedLibs` is not used anymore. That seems very fishy to me.

Reply:
All the unpacked libs are in the same directory so the `glob` looks in this case for `native/*/scala-native` so the individual paths are not needed. I could possibly build the pattern for the `unpackedLibs` such as the following:
`{"dir-0", "dir-1", ...}` - perhaps that is better. I would like to avoid traversing the the directory more than once.